src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -560,39 +561,41 @@ private static InstanceConfiguration instanceConfiguration(
       boolean oneShot,
       String template) {
     InstanceConfiguration ic =
-        new InstanceConfiguration(
-            NAME_PREFIX,
-            REGION,
-            ZONE,
-            MACHINE_TYPE,
-            numExecutors,
-            startupScript,
-            PREEMPTIBLE,
-            MIN_CPU_PLATFORM,
-            labels,
-            CONFIG_DESC,
-            BOOT_DISK_TYPE,
-            BOOT_DISK_AUTODELETE,
-            BOOT_DISK_IMAGE_NAME,
-            BOOT_DISK_PROJECT_ID,
-            BOOT_DISK_SIZE_GB_STR,
-            false,
-            "",
-            "",
-            createSnapshot,
-            null,
-            new AutofilledNetworkConfiguration(NETWORK_NAME, SUBNETWORK_NAME),
-            EXTERNAL_ADDR,
-            false,
-            NETWORK_TAGS,
-            SERVICE_ACCOUNT_EMAIL,
-            RETENTION_TIME_MINUTES_STR,
-            LAUNCH_TIMEOUT_SECONDS_STR,
-            NODE_MODE,
-            new AcceleratorConfiguration(ACCELERATOR_NAME, ACCELERATOR_COUNT),
-            RUN_AS_USER,
-            oneShot,
-            template);
+        new InstanceConfiguration.Builder()
```

Comment:
This builder looks very similar to the Instance builder included in the compute client libs: https://googleapis.github.io/google-cloud-java/google-cloud-clients/apidocs/index.html?com/google/cloud/compute/v1/package-summary.html

We'll need to evaluate if it's worthwhile for us to maintain duplicate versions of these models which already exists.

Reply:
Not necessarily in this PR, but something for us to consider long term.